cmd/web/main.go

```diff
@@ -66,6 +65,26 @@ func main() {
 		log.Fatal().Err(err).Send()
 	}
 
+	conn, err := amqp.Dial(cfg.rabbitMQConnString)
+	if err != nil {
+		log.Fatal().Err(err).Send()
+	}
+	ch, err := conn.Channel()
+	if err != nil {
+		log.Fatal().Err(err).Send()
+	}
+
+	queue, err := ch.QueueDeclare(
+		"emails",
+		false,
+		false,
+		false,
+		false,
+		nil,
+	)
+	if err != nil {
+		log.Fatal().Err(err).Send()
+	}
```

Comment:
This code is duplicated in `/mailer/main.go`. Looks like it common action for all services (present and future) so, it make sense to extract it and re-use

Reply:
Extracted for re-use.